examples/medplum-provider/src/hooks/useDebouncedUpdateResource.ts

```diff
@@ -14,13 +14,17 @@ export const DEFAULT_SAVE_TIMEOUT_MS = 500;
 export function useDebouncedUpdateResource<T extends Resource>(
   medplum: MedplumClient,
   timeoutMs: number = DEFAULT_SAVE_TIMEOUT_MS
-): (resourcePayload: T) => Promise<T> {
-  const debouncedCallback = useDebouncedCallback(async (resourcePayload: T): Promise<T> => {
-    return (await medplum.updateResource(resourcePayload)) as T;
+): (resourcePayload: T) => void {
```

Comment:
Hm... I took a look, and this is used elsewhere such as `BillingPage`.... Will this affect behavior there, if this method no longer waits for completion?

At minimum, we should remove `await` from all call sites if the function is no longer `async`.